src/notification-logs/entities/notification-log.entity.ts

```
@@ -0,0 +1,38 @@
+    message: string;
+
+    @Column({ name: 'is_read', default: false })
+    isRead: boolean;
+
+    @Column({ name: 'scheduled_at' })
+    scheduledAt: Date;
+
+    @Column({ name: 'sent_at', nullable: true })
+    sentAt: Date | null;
+
+    @Column({ name: 'device_id' })
```

ESLint

[error] Delete `··`

(prettier/prettier)